src/components/SnazzyButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import { snazzyButtonStyle } from './helpers';
+interface ButtonProps {
+  children: React.ReactNode;
+}
+
+const styles = {
+  snazzyButton: {
+
+    backgroundColor: 'blue',
+    color: 'white',
+    padding: '10px 20px',
+    borderRadius: '5px',
+
```

Comment:
This styling is not needed. You need to using the snazzyButtonStyle you just imported. This is where that goes.